### DISCUSSION TITLE
Error during compiling

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#4928] [General]
I downloaded the latest version today (Jan 16, 2023) and I am getting the following error at npm run start

![error](https://user-images.githubusercontent.com/83007711/212667089-3dcb5111-e476-487c-8528-a9fcf761015b.JPG)

Compile log: [compile.log](https://github.com/iTwin/itwinjs-core/files/10425285/compile.log)

Please advise what I might be doing wrong, I started from top multiple times, but getting same error.

Ovais

**Comment by aruniverse** [comment]
Hi. Please upgrade to 3.5.3, the issue you are seeing should be resolved with the latest patch.

**Reply by ovaisa** [reply]
Can you please guide which repository I should be using.  I downloaded from the location in the instructions.

npx create-react-app your-app-name --template @itwin/web-viewer --scripts-version @bentley/react-scripts

Kindly provide instructions how to upgrade to 3.5.3.  Should I just modify package.json or there are additional requirements to upgrade?

**Reply by ovaisa** [reply]
Tried changing package.json (and lock.json) to 3..5.3, getting same error.  Please guide on upgrading.

**Reply by aruniverse** [reply]
After updating the package.json, did you do an install (npm run install)?

**Reply by ovaisa** [reply]
Yes, and reconfirmed again now by running npm install

**Reply by aruniverse** [reply]
And just to confirm, you restarted the dev-server (killed the start command and reran it) right?

**Reply by aruniverse** [reply]
Apologies, looks like we had an issue with our pipelines which prevented the 3.5.3 publish; which has now been published as of 14 mins ago. https://www.npmjs.com/package/@itwin/core-bentley/v/3.5.3 Please try pulling this version in.

**Comment by ovaisa** [comment]
Yes

**Comment by ovaisa** [comment]
Can I get path of the latest patched repository (3.5.3) so that I can do a clean download and build

Thanks,

Ovais

**Reply by aruniverse** [reply]
Can you post the most up to date stack trace you see in your shell after upgrading to 3.5.3?

Another way to fix this is to downgrade @bentley/react-scripts from 5 to `"^4.0.9"`
